ecoscope_workflows/tasks/results/_ecoplot.py

```diff
@@ -1,12 +1,37 @@
 from typing import Annotated, Literal
 
-from pydantic import Field
+from pydantic import BaseModel, Field
 from pydantic.json_schema import SkipJsonSchema
 
 from ecoscope_workflows.annotations import DataFrame, JsonSerializableDataFrameModel
 from ecoscope_workflows.decorators import task
 
 
+class LineStyle(BaseModel):
+    color: str | SkipJsonSchema[None] = None
+
+
+class PlotStyle(BaseModel):
+    xperiodalignment: str | SkipJsonSchema[None] = None
+    marker_colors: list[str] | SkipJsonSchema[None] = None
+    textinfo: str | SkipJsonSchema[None] = None
+    line: LineStyle | SkipJsonSchema[None] = None
+
+
+class PlotCategoryStyle(BaseModel):
+    marker_color: str | SkipJsonSchema[None] = None
+
+
+class GroupedPlotStyle(BaseModel):
+    category: str
+    plot_style: PlotCategoryStyle
+
+
+class LayoutStyle(BaseModel):
+    font_color: str | SkipJsonSchema[None] = None
+    font_style: str | SkipJsonSchema[None] = None
+
```

Comment:
```suggestion
class LayoutStyle(BaseModel):
    font_color: str | SkipJsonSchema[None] = None
    font_style: str | SkipJsonSchema[None] = None
    plot_bgcolor: str | SkipJsonSchema[None] = None
    showlegend: bool | SkipJsonSchema[None] = None


class BarLayoutStyle(LayoutStyle):
    bargap: Annotated[float, ValueRange(0.0, 1.0)] | SkipJsonSchema[None] = None
    bargroupgap: Annotated[float, ValueRange(0.0, 1.0)] | SkipJsonSchema[None] = None
```
Just some extra style params that may be useful